### ISSUE TITLE
getter and function `enabled` have the same name

### ISSUE THREAD

**Issue by nt4f04uNd** [creativecreatorormaybenot/wakelock#77]
![image](https://user-images.githubusercontent.com/39104740/104212008-cb95c980-5445-11eb-9d78-2e14a3e8d88d.png)

cheking the source code i can see two getters the do the same, but you deprected another valid one (and in latest release even removed it at all)
why did you added enabled getter that causes this exception, did you even test your change?

i'm using wakelock: ^0.2.1+1

<details>
<summary>flutter doctor -v</summary>

```
[✓] Flutter (Channel master, 1.24.0-8.0.pre.374, on Microsoft Windows [Version 10.0.19041.685], locale ru-RU)
    • Flutter version 1.24.0-8.0.pre.374 at c:\dev\src\flutter
    • Framework revision 183f0e797a (7 weeks ago), 2020-11-26 19:12:28 +0100
    • Engine revision 20caf54969
    • Dart version 2.12.0 (build 2.12.0-76.0.dev)

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at C:\Users\danya\AppData\Local\Android\sdk
    • Platform android-30, build-tools 30.0.2
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    • All Android licenses accepted.

[✓] Android Studio (version 4.0)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin version 50.0.1
    • Dart plugin version 193.7547
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)

[✓] VS Code (version 1.52.1)
    • VS Code at C:\Users\danya\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.18.1

[✓] Connected device (2 available)
    • Redmi Note 5 (mobile)   • 40c9f14       • android-arm64 • Android 9 (API 28)
    • sdk gphone x86 (mobile) • emulator-5554 • android-x86   • Android 11 (API 30) (emulator)

• No issues found!
```
</details>

**Comment by creativecreatorormaybenot** [comment]
Hi @nt4f04uNd. Thanks for opening the issue 🙂 

Of course I tested the code when I deployed it. The package is also unit and integration tested and used in numerous production apps.

There **has never been** an `enabled()` function. You cannot have a getter and a function member of the same name - that would not combine.

What you are looking for is either `Wakelock.enable();` (without the `d`) or `Wakelock.enabled;`. There was a `Wakelock.isEnabled;` getter before, however, I removed it because the naming was not good.

Here are the docs entries:

* https://pub.dev/documentation/wakelock/latest/wakelock/Wakelock/enabled.html
* https://pub.dev/documentation/wakelock/latest/wakelock/Wakelock/enable.html
